.circleci/config.yml

```diff
@@ -254,6 +254,20 @@ jobs:
           command: |
             forge --version
             forge test -vvv
+  task_test:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - run:
+          name: task test
+          command: |
+            forge --version
+            forge build
+            forge script src/fps/example/template/GasConfigTemplate.sol --sig "run(string)" src/fps/example/task-00/mainnetConfig.toml --rpc-url mainnet -vvv
```

Comment:
These will need to change now based on new directory structures. When we have the just file, this stuff can be condensed into a single command like `just test-templates`.